nodeup/pkg/model/kube_apiserver.go

```diff
@@ -638,11 +638,17 @@ func (b *KubeAPIServerBuilder) buildPod(kubeAPIServer *kops.KubeAPIServerConfig)
 		container.Args = append(container.Args, sortedStrings(flags)...)
 	} else {
 		container.Command = []string{"/usr/local/bin/kube-apiserver"}
-		container.Args = append(
-			sortedStrings(flags),
-			"--logtostderr=false", // https://github.com/kubernetes/klog/issues/60
-			"--alsologtostderr",
-			"--log-file=/var/log/kube-apiserver.log")
+		if kubeAPIServer.LogFormat != "" && kubeAPIServer.LogFormat != "text" {
```

Comment:
Should this be done for `b.IsKubernetesGTE("1.23")` too?

Reply:
No, when Kubernetes version is greater than 1.23, we use `go-runner` and we don't specify these logging options as kube-apiserver's flags. So it is no problem.